spec/forms/two_factor_setup_form_spec.rb

```diff
@@ -81,10 +106,11 @@
         }
         result = instance_double(FormResponse)
 
+        params[:phone] = ''
```

Comment:
should we do this via `let(:phone) { '' }` inside the context? Ditto for the other specs in the file